core/backend/src/test/integration/ChangedElements.test.ts

```diff
@@ -2,63 +2,57 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-import { DbResult } from "@bentley/bentleyjs-core";
-import { ChangedElements } from "@bentley/imodeljs-common";
+import { DbResult, GuidString } from "@bentley/bentleyjs-core";
+import { IModelError } from "@bentley/imodeljs-common";
 import { TestUsers, TestUtility } from "@bentley/oidc-signin-tool";
 import { assert } from "chai";
 import { ChangedElementsManager } from "../../ChangedElementsManager";
 import { AuthorizedBackendRequestContext, BriefcaseManager, ChangedElementsDb, IModelHost, IModelJsFs, ProcessChangesetOptions } from "../../imodeljs-backend";
-import { IModelTestUtils, TestIModelInfo } from "../IModelTestUtils";
+import { IModelTestUtils } from "../IModelTestUtils";
 import { HubUtility } from "./HubUtility";
 
-function setupTest(iModelId: string): void {
-  const cacheFilePath: string = BriefcaseManager.getChangeCachePathName(iModelId);
-  if (IModelJsFs.existsSync(cacheFilePath))
-    IModelJsFs.removeSync(cacheFilePath);
-}
-
 describe.skip("ChangedElements (#integration)", () => {
   let requestContext: AuthorizedBackendRequestContext;
-  let testProjectId: string;
-
-  let testIModel: TestIModelInfo;
+  let testContextId: GuidString;
+  let testIModelId: GuidString;
 
   before(async () => {
     requestContext = await TestUtility.getAuthorizedClientRequestContext(TestUsers.regular);
-    testProjectId = await HubUtility.queryProjectIdByName(requestContext, "iModelJsIntegrationTest");
-    testIModel = await IModelTestUtils.getTestModelInfo(requestContext, testProjectId, "ReadOnlyTest");
+
+    testContextId = await HubUtility.getTestContextId(requestContext);
+    requestContext.enter();
+    testIModelId = await HubUtility.getTestIModelId(requestContext, HubUtility.testIModelNames.readOnly);
+    requestContext.enter();
 
     // Purge briefcases that are close to reaching the acquire limit
-    const managerRequestContext = await TestUtility.getAuthorizedClientRequestContext(TestUsers.manager);
-    await HubUtility.purgeAcquiredBriefcases(managerRequestContext, "iModelJsIntegrationTest", "ReadOnlyTest");
+    await HubUtility.purgeAcquiredBriefcasesById(requestContext, testIModelId);

```

Comment:
The regular user was supposed to have limited permissions. Not sure if that changed now - but that was the reason to use manager here.

Reply:
Okay, I thought an individual user had access to free their own briefcases and didn't need someone else to do it for them.  I'll test that out to make sure it works.